src/Indicator.vala

```diff
@@ -126,6 +134,25 @@ public class Session.Indicator : Wingpanel.Indicator {
             }
 
             main_grid.add (suspend);
+
+            try {
+                if (suspend_interface.can_hibernate () == "yes") {
+                    main_grid.add (hibernate);
+                }
+            } catch (GLib.Error e) {
+                warning ("CanHibernate Error: %s", e.message);
+                hibernate.set_sensitive (false);
+            }
+
+            try {
+                if (suspend_interface.can_hybrid_sleep () == "yes") {
+                    main_grid.add (hybrid_sleep);
+                }
+            } catch (GLib.Error e) {
+                warning ("CanHybridSleep Error: %s", e.message);
+                hybrid_sleep.set_sensitive (false);
```

Comment:
Same as above. If the widget is never packed into the container, probably doesn't make a difference if it's sensitive